docs/source/index.rst

```diff
@@ -41,7 +41,7 @@ then be represented as follows:
 
 .. math::
 
-    \phi_{m p} = Y_\ell^m \sum_i c_i e^{-\alpha_i |\phi_{\rm center} - p| ^2}
```

Comment:
`|` typically indicates vector magnitude as is the case here. Unless I am missing something?

Reply:
It's squared so it makes no difference; in this case you usually write it in parentheses because `|...|` makes you think there's a square root involved...